frontend/doc-site/04__Analyze Public Data/4_4__Annotate.mdx

```diff
@@ -0,0 +1,171 @@
+- use this predictive annotation workflow with data generated by 10X platforms
+- use the tissue model which is most closely related to the tissue profiled in your single cell experiment
+
+## Installation of CELLxGENE[annotate]
+
+
+You can install the latest experimental release of CELLxGENE to gain access to the automatic annotation subcommand. It is highly recommended to perform installation in a clean virtual environment (such as conda or venv) with python version 3.8+. Find examples of virtual environment setup using venv and conda below
+
+### venv setup
+
+```
+brew install pyenv  # used by MLflow
+
+virtualenv --python python3.8 /path/to/venv
+
+. /path/to/venv/bin/activate
+
+```
+
+### conda setup
+
+```
+conda create -n cellxgene 'python=3.8'
+
+conda activate cellxgene
+```
+
+### pip install
+Once your virtual environment is installed and activated, install the `annotate` feature using pip using the following command:
+
+`pip install cellxgene[annotate]`
+
+you may also install from our test PyPi release:
+
+`pip install --no-cache-dir --index-url https://test.pypi.org/simple/ --extra-index-url https://pypi.org/simple 'cellxgene[annotate]==1.1.0.dev4'`
+
+You can run the following to test the success of your installation:
+
+`cellxgene annotate -h` or `cellxgene annotate --help`
+
+## CELLxGENE Annotation Workflow
+
+See our quickstart guide to  quickly see if you can run through the pipeline or check out the more detailed sections below for more detailed descriptions of the workflow. Both assume that you have properly set up a clean virtual environment to work from.
+
+### Quickstart
+
+Depending on your environment manager, execute one of the following quickstart commands:
+
+#### venv
+```
+mkdir annotations #create a directory to download data, run annotation pipeline and store annotated object
+
+cd annotations #navigate to newly created directory
+
+wget https://github.com/chanzuckerberg/cellxgene/raw/main/example-dataset/pbmc3k.h5ad # retrieve sample dataset
+
+cellxgene annotate -i pbmc3k.h5ad  -m https://s3.console.aws.amazon.com/s3/object/cellxgene-annotation-public?region=us-west-2&prefix=cell_type/models/immune_ref_20220728145305_venv.zip -o pbmc3kannotated.h5ad # generate annotations
+
+cellxgene launch pbmc3kannotated.h5ad #launch cellxgene explorer
+```
+
+#### conda
+
+```
+mkdir annotations #create a directory to download data, run annotation pipeline and store annotated object
+
+cd annotations #navigate to newly created directory
+
+wget https://github.com/chanzuckerberg/cellxgene/raw/main/example-dataset/pbmc3k.h5ad # retrieve sample dataset
+
+cellxgene annotate -i pbmc3k.h5ad  -m https://cellxgene-annotation-public.s3.us-west-2.amazonaws.com/cell_type/models/immune_ref_20220728145305_conda.zip -o pbmc3kannotated.h5ad # generate annotations
```

Comment:
this command does not run on ubuntu - it throws an error: 
```python3
acarr@acarr-ThinkPad-X1-Carbon-Gen-9:~$ cellxgene annotate -i pbmc3k.h5ad -m https://s3.console.aws.amazon.com/s3/object/cellxgene-annotation-public?region=us-west-2&prefix=cell_type/models/immune_ref_20220728145305_venv.zip --output-h5ad-file pbmc3kannotated.h5ad
[1] 66685
--output-h5ad-file: command not found
```

Swapping h5ad earlier fixes it, but it then errors out due to dependencies. Not sure if the dependencies is user error or not. Will investigate if time permits.

Reply:
Thanks Ambrose - the commands have been updated to address this issue - marking this as resolved for now